src/commands/release/export.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * @license
+ * Copyright 2016-2024 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { commitOrIdArg } from '.';
+import { Flags } from '@oclif/core';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, stripIndent } from '../../utils/lazy';
+import { create } from '@balena/release-bundle';
+import * as fs from 'fs/promises';
+import * as semver from 'balena-semver';
+import { ExpectedError } from '../../errors';
+
+export default class ReleaseExportCmd extends Command {
+	public static description = stripIndent`
+		Exports a release into a file.
+
+        Exporting a release to a file allows you to import an exact 
```

Comment:
Missing an indent here?

Reply:
Just for my better understanding of these commands, you say that this allows you to import an "exact" copy of the original release. Is this true? I remember hearing that simply re-building/pushing a release can yield 2 different outcomes/releases. Does importing a release not build it at all and just use everything from the previously built release exactly as it is?

Reply:
Yes, it pulls the images the selected release from the registry and packages them into the release bundle tar file with the release metadata. The release bundle contains enough information to create the same release in another fleet including notes and tags.

Reply:
Wow, nice!

Reply:
Indentation fixed in latest commit.